guides/v2.1/install-gde/trouble/php/tshoot_php-set.md

```diff
@@ -85,4 +98,6 @@ Multiple templates have syntax for support abstract level on templates (use diff
 
 More information about [asp_tags](http://php.net/manual/en/ini.core.php#ini.asp-tags){:target="_blank"}.
 
-Edit `php.ini` and set `asp_tags = off`. For more information, see [Required PHP settings]({{ page.baseurl }}/install-gde/prereq/php-settings.html).
+Edit `php.ini` and set `asp_tags = off`. For more information, see [Required PHP settings][]
```

Comment:
Add a period at the end of this sentence.

Reply:
Done.